.github/ISSUE_TEMPLATE/unicodetools.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Feedback on the Unicode Tools code and processes
+about: Mostly-Unicode-internal Unicode Tools
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+(Bugs, suggestions, ideas for Unicode Tools code, for its maintainers, their processes, etc.)
+
+(These tools are publicly visible but treated as an implementation detail of PAG.
+Data is published at https://www.unicode.org/Public/ -- feedback: https://www.unicode.org/reporting.html)
```

Comment:
Maybe need to say some more about what this is NOT a little more strongly.  This is not the place to propose data changes, go here.

```suggestion
(Bugs, suggestions, ideas for internal Unicode Tools code, for its maintainers, their processes, etc.)

(These tools are public and open source but are treated as an implementation detail of PAG.
Data is published at https://www.unicode.org/Public/ -- feedback: https://www.unicode.org/reporting.html)
```

Reply:
Change to:

> While those tools are public and open source for reference purposes, they are not a public deliverable. Instead they are used strictly internally by the maintainers of the standard.